support/rbac.go

```diff
@@ -202,3 +202,33 @@ func CreateUserClusterRoleBinding(t Test, userName string, roleName string) *rba
 
 	return rb
 }
+
+func CreateUserNamespacedRoleBinding(t Test, userName string, namespace *corev1.Namespace, roleName string) *rbacv1.RoleBinding {
+	t.T().Helper()
+
+	// Create a RoleBinding to give specified role access to the user for test-namespace
+	roleBinding := &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "rb-",
+			Namespace:    namespace.Name,
+		},
+		RoleRef: rbacv1.RoleRef{
+			Kind:     "ClusterRole",
+			Name:     roleName, // grants admin access
```

Comment:
I guess it is possible to use any role name. Not only admin right? Maybe forgotten comment.

Reply:
Yes, I missed this one.. Updated !!